nuts_and_bolts/main/routes.py

```diff
@@ -1,8 +1,10 @@
-from flask import Blueprint, render_template, redirect, flash, session, abort, request
-
+from flask import Blueprint, render_template, redirect, flash, session, abort, request, url_for
+from .forms import QuestionsForm
```

Comment:
Please change to:
from nuts_and_bolts.forms ...
Explicit imports are better than relative ones.